contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -39,10 +40,32 @@ contract VaultHub is PausableUntilWithRoles {
         uint64 vaultsDataTimestamp;
     }
 
+    /**
+     * @notice Latest reported totalValue and inOutDelta
+     * @custom:totalValue Aggregated validator balances plus the balance of `StakingVault`
+     * @custom:inOutDelta Net difference between ether funded and withdrawn from `StakingVault`
+     */
+    struct Report {
+        uint128 totalValue;
+        int128 inOutDelta;
+        uint64 timestamp;
+    }
+
+    // todo: optimize storage layout
     struct VaultSocket {
         // ### 1st slot
         /// @notice vault address
         address vault;
+        /// @notice vault manager address
+        address manager;
+        /// @notice latest report
+        Report report;
+        /// @notice amount of ETH that is locked on the vault and cannot be withdrawn by manager
+        uint128 locked;
+        /// @notice net difference between ether funded and withdrawn from the vault
+        int128 inOutDelta;
+        /// @notice true if connection not confirmed by the report
+        bool pendingConnect;
```

Comment:
:warning: probably we don't need this field at all

and we won't need `cancelPendingConnect`